proprietary/tokens/src/components/ams/radio.tokens.json

```diff
@@ -11,65 +11,65 @@
       "text-decoration-thickness": { "value": "{ams.link-appearance.text-decoration-thickness}" },
       "text-underline-offset": { "value": "{ams.link-appearance.text-underline-offset}" },
       "checked-indicator": {
-        "fill": { "value": "{ams.color.primary-blue}" },
+        "fill": { "value": "{ams.color.interactive.default}" },
         "disabled": {
-          "fill": { "value": "{ams.color.neutral-grey3}" }
+          "fill": { "value": "{ams.color.interactive.disabled}" }
         },
         "disabled-invalid": {
-          "fill": { "value": "{ams.color.neutral-grey3}" },
+          "fill": { "value": "{ams.color.interactive.disabled}" },
           "hover": {
-            "fill": { "value": "{ams.color.neutral-grey3}" }
+            "fill": { "value": "{ams.color.interactive.disabled}" }
           }
         },
         "hover": {
-          "fill": { "value": "{ams.color.dark-blue}" }
+          "fill": { "value": "{ams.color.interactive.hover}" }
         },
         "invalid": {
-          "fill": { "value": "{ams.color.primary-red}" },
+          "fill": { "value": "{ams.color.interactive.invalid.default}" },
           "hover": {
-            "fill": { "value": "{ams.color.primary-red}" }
+            "fill": { "value": "{ams.color.interactive.invalid.hover}" }
           }
         }
       },
       "hover-indicator": {
         "stroke-width": { "value": "3px" },
         "hover": {
-          "stroke": { "value": "{ams.color.primary-blue}" }
+          "stroke": { "value": "{ams.color.interactive.default}" }
         },
         "invalid": {
           "hover": {
-            "stroke": { "value": "{ams.color.primary-red}" }
+            "stroke": { "value": "{ams.color.interactive.hover}" }
```

Comment:
This should be `interactive.invalid.hover`

Reply:
Indeed!